spec/activerecord-bitemporal/bitemporal_spec.rb

```diff
@@ -27,10 +27,13 @@
         it {
           is_expected.to have_attributes(
             bitemporal_id: subject.id,
+            # changes: be_empty,
```

Comment:
Perhaps the expected is `be_empty`, but unfortunately that is not the case at this time. This will be fixed in another PR.